### ISSUE TITLE
Not working in fedora 37

### ISSUE THREAD

**Issue by adetabrani** [vagnum08/cpupower-gui#103]
**Describe the bug**
I've done a manual installation along with installing the required dependencies, but it doesn't work when I run it,
i just get an error message when i run it
![1679133197_Mar18_16:53:17](https://user-images.githubusercontent.com/47938004/226098619-236f3b92-0b5b-4c66-98f8-2a6b8bc55aca.png)

**To Reproduce**
Steps to reproduce the behaviour:
1. Type command  'cpupower-gui' on terminal
2. See error

**Expected behaviour**
The gui application should open

**Setup (please complete the following information):**
 - Distro: Fedora 37 workstation
 - Package version: 1.0.0
 - Installed via: source

**Comment by adetabrani** [comment]
> From what I can see there is an error with D-Bus. the messaging service used to communicate between the gui and the cpu-helper.
> 
> Did you use the master branch for install or the latest tag?

it seems yes, I downloaded the package using git clone, but when I try to build using the latest package tag from [here](https://github.com/vagnum08/cpupower-gui/releases/tag/v1.0.0) , I can't build, see the error below
![1679738656_Mar25_17:04:16](https://user-images.githubusercontent.com/47938004/227710653-a43655ae-5744-4698-8fe0-d9fd96dea8ea.png)

**Comment by vagnum08** [comment]
Sorry for the late response.

This is an issue with newer versions of meson that deprecated an option.
To build version 1.0.0 you have to edit the `meson.build` file that is in  `data` folder.
Remove line 15 (that says `'desktop',`).

When you have removed the line it will build without any issues.

Alternatively, you can build from the latest commit that has this issue fixed.

Let me know if you have any issues.

**Comment by adetabrani** [comment]
![1684167185_May15_23:13:05](https://github.com/vagnum08/cpupower-gui/assets/47938004/5cf7419c-3794-4268-855c-fdb6292d73d1)

Already installed, but I still get these error logs when I try to run it

**Comment by vagnum08** [comment]
You have to enable and start the cpupower-helper service from systemd

**Comment by adetabrani** [comment]
![1684303233_May17_13:00:33](https://github.com/vagnum08/cpupower-gui/assets/47938004/083a3ebf-bf93-4b2e-89fd-4938c3c2c21c)

Can't start the service

**Comment by adetabrani** [comment]
just generates that log message

**Comment by k2helix** [comment]
This error keeps happening and makes cpupower-gui unusable

**Comment by vagnum08** [comment]
I will try to boot fedora 37 and see what is happening.

**Comment by k2helix** [comment]
> I will try to boot fedora 37 and see what is happening.

I was in Fedora 38, which is the latest

**Comment by christophehenry** [comment]
@vagnum08 I have the same error. Here is the stacktrace:

```
# /usr/local/lib64/cpupower-gui/cpupower-gui-helper
Traceback (most recent call last):
  File "/usr/local/lib64/cpupower-gui/cpupower-gui-helper", line 313, in <module>
    dbus_service = CpupowerGui_DBus(loop)
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib64/cpupower-gui/cpupower-gui-helper", line 58, in __init__
    bus_name = dbus.service.BusName("org.rnd2.cpupower_gui.helper", bus=self.bus)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/site-packages/dbus/service.py", line 134, in __new__
    retval = bus.request_name(name, name_flags)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/site-packages/dbus/bus.py", line 299, in request_name
    return self.call_blocking(BUS_DAEMON_NAME, BUS_DAEMON_PATH,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/site-packages/dbus/connection.py", line 634, in call_blocking
    reply_message = self.send_message_with_reply_and_block(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.AccessDenied: Request to own name refused by policy
```

**Comment by vagnum08** [comment]
I booted a live-usb with Fedora 38 today and I did the following:

Opened the terminal and run `dnf config-manager --add-repo https://download.opensuse.org/repositories/multimedia:proaudio/Fedora_38/multimedia:proaudio.repo` to add the repo from OBS (by Geekos DAW project)

Then run `dnf install cpupower-gui`.

After the installation, I run `cpupower-gui` in the same terminal and it worked.

No errors, nothing.
Tried from the app icon and did the same.

I would suggest to uninstall the version you have and install the one from the repo above (as a bonus, the version on that repo is the latest from git, not 1.0.0).

Let me know if it works.
